proxy/src/lib.rs

```diff
@@ -22,6 +22,8 @@ pub mod schema_config_provider;
 mod util;
 mod write;
 
+pub const FORWARDED: &str = "forwarded";
```

Comment:
How about this name?
```suggestion
pub const FORWARDED_FROM: &str = "Forwarded-From";
```
And move this definition to `the forward.rs`.